templates/incl-editor-and-output.html

```diff
@@ -85,9 +85,19 @@
       </div>
     </div>
     <div class="flex flex-col order-3" id="code_output" style="height: 22rem;">
-      <!-- tabindex=0 makes the div focusable -->
-      <div id="output" tabindex=0 class="flex-1 rounded p-2 px-3 bg-gray-900 color-white w-full text-lg overflow-auto" style="min-height: 3rem"></div>
-      <div id="turtlecanvas" class="flex-0 ltr:pl-4 rtl:pr-4"></div>
+  <!-- tabindex=0 makes the div focusable -->
+        <div id="output" tabindex=0 class="flex-1 rounded p-2 px-3 bg-gray-900 color-white w-full text-lg" style="position:relative"
+             style="min-height: 3rem">
+             <div class="dropdown" style="height: 30px; width: 62px;margin-left: 88%; margin-top: -54px;">
+                 <button id="variable_button" onclick="hedyApp.showVariableView()" class="green-btn mt-4 inline-block absolute z-10 mx-2 mt-2 mb-0 text-white flex-0" style="cursor: pointer; right:0rem; top:0rem; background-color: #3182ce; border-bottom-width: 0px; font-family: Segoe UI Emoji; padding: 1.5px 8.2px 1.5px 8.2px;"><span style="float:left">🏷<svg style="float:right" class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round"  d="M19 9l-7 7-7-7"></path></svg></span></button>
+             </div>
+            <div id="variables" class="mx-auto w-64 invisible" style="display: block; padding: 0.5rem; position:absolute; z-index:9; border-width: 1px; background-color: #3182ce; border-width: 1px; width: fit-content; height: 10.5rem; color: white; border-radius: 0.5rem; top: 44px; right: 6.5px;" onclick="hedyApp.show_variables()" >
+  <div id="variables_header" style="z-index:10; margin-top:-3.75rem; text-align: center; font-weight: bold;">{{ui.variables}}</div>
+  <ul class="variable-list hidden" style="list-style-type:none; margin-left: 0; margin-top: -3.8rem; line-height: 125%; font-size:1rem;">
```

Comment:
This would also be nice to be styled a bit "cleaner". The code will be hard to maintain if we ever want to change the UI of our code page.